tasks/section1.yml

```diff
@@ -1220,7 +1220,7 @@ print $NF }'"
       - rule_1.7.1.2
 
 - name: "NOTSCORED | 1.7.1.3 | AUDIT | Ensure remote login warning banner is configured properly"
-  command: /bin/true
+  command: egrep '(\\v|\\r|\\m|\\s)' /etc/issue.net
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*